### PR TITLE
Updated copyright header for the files that are changed in 2021.1

### PR DESCRIPTION
**Pull request by sureshreddyavula**
Updated the copyright year and company info from Xilinx to AMD. Tried to match with the existing format used in other files of the same module.

### CODE REVIEW

**Comment by sureshreddyavula** [comment]
@rbramand-xilinx, @chvamshi-xilinx I updated copyright info for the files that are changed by us. This is done as per the legal team request for U30 GA3.1 release. Let me know if this doesn't apply to XRT.

**Comment by sureshreddyavula** [comment]
@rbramand-xilinx, @chvamshi-xilinx Can you review this PR and let us know your comments?

File: src/runtime_src/core/edge/drm/zocl/zocl_drv.c [hunk 1]
```diff
@@ -4,6 +4,7 @@
  * OpenCL accelerators.
  *
  * Copyright (C) 2016-2021 Xilinx, Inc. All rights reserved.
+ * Copyright (C) 2021-2024 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (rbramand-xilinx) [inline src/runtime_src/core/edge/drm/zocl/zocl_drv.c:7]: It should be 2016 - 2022 Xilinx and 2022 - 2024 Advanced Micro Devices

File: src/runtime_src/core/pcie/driver/aws/kernel/mgmt/mgmt-core.h [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /*
  *  Copyright (C) 2017-2018, Xilinx Inc
+ *  Copyright (C) 2018-2024 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (rbramand-xilinx) [inline src/runtime_src/core/pcie/driver/aws/kernel/mgmt/mgmt-core.h:3]: same here, 2017 - 2022 Xilinx and 2022 - 2024 Advanced Micro Devices

File: src/runtime_src/core/pcie/driver/linux/xocl/lib/libqdma/qdma_request.c [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
  * This file is part of the Xilinx DMA IP Core driver for Linux
  *
- * Copyright (c) 2017-present,  Xilinx, Inc.
+ * Copyright (c) 2017-present,  Advanced Micro Devices, Inc.
```
Comment (rbramand-xilinx) [inline src/runtime_src/core/pcie/driver/linux/xocl/lib/libqdma/qdma_request.c:4]: Please follow same approach in all the files as mentioned above, * - 2022 Xilinx and 2022 - 2024 Advanced Micro Devices

**Comment by sureshreddyavula** [comment]
@rbramand-xilinx, @chvamshi-xilinx Addressed the review comments. Can you please review?

**Review by rbramand-xilinx — APPROVED** [review]

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
Merging as this change required for a release and build is failing due to infra issue.

**Pull request merged**